signer/src/message.rs

```diff
@@ -404,6 +417,7 @@ impl wsts::net::Signable for Payload {
             Self::StacksTransactionSignature(msg) => msg.hash(hasher),
             Self::SweepTransactionInfo(msg) => msg.hash(hasher),
             Self::BitcoinPreSignRequest(msg) => msg.hash(hasher),
+            Self::BitcoinPreSignAck(_) => hasher.update("SIGNER_BITCOIN_PRE_SIGN_ACK"),
```

Comment:
suggestion: just for consistency's sake I would probably have implemented this in the same way as the others, even if it's a little more code.

Reply:
fixed in https://github.com/stacks-network/sbtc/pull/956/commits/d86363dbb1f0164a5944ab6ac71c399390691dcb